src/core_codemods/remove_module_global.py

```diff
@@ -0,0 +1,22 @@
+import libcst as cst
+from libcst.metadata import GlobalScope, ScopeProvider
+from typing import Union
+from codemodder.codemods.api import BaseCodemod, ReviewGuidance
+from codemodder.codemods.utils_mixin import NameResolutionMixin
+
+
+class RemoveModuleGlobal(BaseCodemod, NameResolutionMixin):
+    NAME = "remove-module-global"
+    SUMMARY = "Remove Module-level Global Call"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_WITHOUT_REVIEW
+    DESCRIPTION = "Remove Lines with `global` keyword at Module Level"
+    REFERENCES: list = []
+
+    def leave_Global(
+        self, original_node: cst.Global, _
+    ) -> Union[cst.Global, cst.RemovalSentinel,]:
```

Comment:
It's lacking the line include/exclude check.

Reply:
so glad you're realizing this bc I clearly definitely forgot it multiple times.

Reply:
I did too, which is how we noticed it in the first place 😅